importers/djornl/parser.py

```diff
@@ -364,6 +373,14 @@ def store_parsed_edge_data(self, datum):
             # report non-matching data
             return f"duplicate data for edge {edge_key}"
 
+        # create a unique key for the DB for this record
+        datum["_key"] = "__".join(
```

Comment:
moving `_key` generation out of the remapping stage and into the storage stage to ensure that all values have already been transformed prior to creating the `_key`